pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -239,6 +239,8 @@ const (
 	LabelScheduler = "scheduler"
 	// LabelExtensionProjectRole is a constant for a label value for extension project roles
 	LabelExtensionProjectRole = "extension-project-role"
+	// LabelResourceManager is a constant for a label value for the gardener-resource-manager.
+	LabelResourceManager = "gardener-resource-manager"
```

Comment:
Why is this here? It should be better in the `resourcemanager` package, or?

Reply:
I deleted it completely I do not think is used at all, or?